File: Código Fuente/Microbit/JavaScript/Registro.js
let Registro = 0
input.onButtonPressed(Button.A, function () {
    Registro += 1
    basic.showNumber(Registro)
})
input.onButtonPressed(Button.AB, function () {
    Registro = 0
    basic.showNumber(Registro)
})
input.onButtonPressed(Button.B, function () {
    Registro += -1
    basic.showNumber(Registro)
})

File: Código Fuente/Microbit/Python/Registro.py
Registro = 0

def on_button_pressed_a():
    global Registro
    Registro += 1
    basic.show_number(Registro)
input.on_button_pressed(Button.A, on_button_pressed_a)

def on_button_pressed_ab():
    global Registro
    Registro = 0
    basic.show_number(Registro)
input.on_button_pressed(Button.AB, on_button_pressed_ab)

def on_button_pressed_b():
    global Registro
    Registro += -1
    basic.show_number(Registro)
input.on_button_pressed(Button.B, on_button_pressed_b)
